conf/jasminegraph-server.properties

```diff
@@ -76,9 +70,15 @@ org.jasminegraph.fl.organization.file=/home/ubuntu/software/jasminegraph/conf/ho
 #--------------------------------------------------------------------------------
 
 #Scheduler Specific Parameters
-org.jasminegraph.scheduler.enabled=false
+org.jasminegraph.scheduler.enabled=true
 #PerformanceCollector Scheduler Timing. Run once every 120 seconds
-org.jasminegraph.scheduler.performancecollector.timing=120
+org.jasminegraph.scheduler.performancecollector.timing=30
+
+#--------------------------------------------------------------------------------
+#PerformanceCollector
+#--------------------------------------------------------------------------------
+org.jasminegraph.collector.pushgateway=http://192.168.43.135:9091/
```

Comment:
Are these IP addresses constant? If not are there additional setup steps involved in setting the IP addresses?

Reply:
This is only used in Docker mode. It should be set to the pushgateway container's IP address before running

Reply:
> Is there a way to automate this? It will be tricky configuration parameter.

There is no easy way to automate this as the configuration file is already included in the docker image.

Reply:
Lets not change the `docker run` command. Instead, lets update the details of the step of adding this configuration parameter in the JasmineGraph documentation.